lms/static/scripts/frontend_apps/components/GroupConfigSelector.js

```diff
@@ -73,13 +80,115 @@ function GroupSelect({ busy, groupSets, onInput, selectedGroupSetId }) {
   );
 }
 
+/**
+ *
+ *  @typedef GroupSelectHeaderProps
```

Comment:
```suggestion
/**
 * @typedef GroupSelectHeaderProps
```

Small formatting correction here and for several other cases. Our formatter unfortunately doesn't touch JSDoc. When we switch this over to TypeScript in future, that problem will go away.